rcjaRegistration/templates/common/regionsFieldJS.html

```diff
@@ -0,0 +1,49 @@
+<script>
+  var regionLookups = {
+    {% for state in regionsLookup %}
+      '{{ state.id }}': [
+        {% for region in state.regions %}
+          {'id': {{ region.id }}, 'name': '{{ region.name }}'},
+        {% endfor %}
+      ],
+    {% endfor %}
+  };
+
+  $(document).ready(function () {
+    homeStateeHandler($('#{{ stateFieldID }}').val());
```

Comment:
Already noted over the phone, but we are mixing ES6 and JQuery here.

The ES6 way to do this would be 

` document.getElementById('{{stateFieldID}}.value`

Reply:
Importing jquery for fomantic anyway, not best practice but not a major issue. Someone else can refactor this with the rest of the frontend.